src/components/onboarding.tsx

```diff
@@ -52,6 +57,12 @@ export default function Onboarding() {
 		fileForImport,
 	} = useAddSite();
 	const [ fileError, setFileError ] = useState( '' );
+	const { importState } = useImportExport();
+	const { data } = useSiteDetails();
+
+	const isAnySiteProcessing = data.some(
+		( site ) => site.isAddingSite || importState[ site.id ]?.isNewSite
+	);
```

Comment:
We use this logic in a couple of places so we could probably move it into a separate hook. Happy to create a follow -up PR!